src/content/pt/2019/ecommerce.md

```diff
@@ -0,0 +1,639 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=1341906463&format=interactive"
+  )
+}}
+
+[O First Contentful Paint](./performance#first-contentful-paint) mede o tempo que leva desde a navegação até que o conteúdo, como texto ou imagem, seja exibido pela primeira vez. Neste contexto, **rápido** significa FCP em menos de um segundo, **lento** significa FCP em 3 segundos ou mais e **moderado** é tudo o que está entre os dois. Observe que o conteúdo e o código de terceiros podem ter um impacto significativo no FCP.
+
+Todas as seis principais plataformas de comércio eletrônico têm pior FCP no celular do que no desktop: menos rápido e mais lento. Observe que o FCP é afetado pela capacidade do dispositivo (capacidade de processamento, memória, etc.), bem como pela conectividade.
+
+Precisamos estabelecer por que o FCP é pior no celular do que no desktop. Quais são as causas: conectividade e/ou capacidade do dispositivo ou outra coisa?
+
+## Pontuações do Progressive Web App (PWA)
+
+Consulte também o [capítulo PWA](./pwa) para obter mais informações sobre este tópico, além de apenas sites de comércio eletrônico.
+
+{{ figure_markup(
+  image="fig22.png",
+  caption="Distribution of Lighthouse PWA category scores for mobile ecommerce pages.",
+  description="Distribution of Lighthouse's PWA category scores for ecommerce pages. On a scale of 0 (failing) to 1 (perfect), 40% of pages get a score of 0.33. 1% of pages get a score above 0.6.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=1148584930&format=interactive"
+  )
+}}
+
+Mais de 60% das páginas iniciais em plataformas de comércio eletrônico obtêm uma <a hreflang="en" href="https://developers.google.com/web/ilt/pwa/lighthouse-pwa-analysis-tool">pontuação Lighthouse PWA</a> entre 0.25 e 0.35. Menos de 20% das páginas iniciais em plataformas de comércio eletrônico obtêm uma pontuação de mais de 0.5 e menos de 1% das páginas iniciais obtêm uma pontuação superior a 0.6.
+
+O Lighthouse retorna uma pontuação do Progressive Web App (PWA) entre 0 e 1. 0 é a pior pontuação possível e 1 é a melhor. As auditorias do PWA são baseadas na <a hreflang="en" href="https://developers.google.com/web/progressive-web-apps/checklist"> Lista de Verificação de Base do PWA</a>, which lists 14 requirements. que lista 14 requisitos. O Lighthouse automatizou auditorias para 11 dos 14 requisitos. Os 3 restantes só podem ser testados manualmente. Cada uma das 11 auditorias PWA automatizadas são ponderadas igualmente, de modo que cada uma contribui com aproximadamente 9 pontos para sua pontuação PWA.
```

Comment:
There is a non-translated phrase here ("which lists 14 requirements")

Reply:
> There is a non-translated phrase here ("which lists 14 requirements")

fixed up